proteus/mprans/BoundaryConditions.py

```diff
@@ -357,10 +358,21 @@ def setAtmosphere(self, orientation=None, vof_air=1.):
         if orientation[2] == 1. or orientation[2] == -1.:
             self.w_diffusive.setConstantBC(0.)
             self.ws_diffusive.setConstantBC(0.)
-        self.k_dirichlet.setConstantBC(1e-30)
-        self.k_diffusive.setConstantBC(0.)
-        self.dissipation_diffusive.setConstantBC(0.)
-
+        
+        if kInflow is not None:
+            self.k_dirichlet.setConstantBC(kInflow)
+            self.k_diffusive.setConstantBC(0.)
+        else:
+            self.k_dirichlet.setConstantBC(1e-20)
+            self.k_diffusive.setConstantBC(0.)
+            logEvent("WARNING: Dirichlet condition for k in "+str(self.BC_type)+" has not been set")
```

Comment:
As these are the default (in the else statement), it might not be necessary to reset them here (self.reset() is called at the beginning of this function). For the warning, as it is now, it will appear even if the k-eps model is not used for the simulation when setting this BC.